packages/typespec-test/test/eventgrid_modular/generated/typespec-ts/README.md

```diff
@@ -1,56 +1,37 @@
 # Azure.Messaging.EventGrid REST client library for JavaScript
 
-Azure Messaging EventGrid Client
-
-**Please rely heavily on our [REST client docs](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/rest-clients.md) to use this library**
+This package contains an isomorphic SDK (runs both in Node.js and in browsers) for Azure.Messaging.EventGrid REST client.
 
-Key links:
+Azure Messaging EventGrid Client
 
-- [Package (NPM)](https://www.npmjs.com/package/@msinternal/eventgrid)
+[Package (NPM)](https://www.npmjs.com/package/@msinternal/eventgrid) |
```

Comment:
where are the other key links? I remember there's four of them?

Reply:
in main, it only has one link https://github.com/Azure/autorest.typescript/blob/main/packages/typespec-test/test/eventgrid_modular/generated/typespec-ts/README.md